package.json

```diff
@@ -7,8 +7,9 @@
         "dev": "vite",
         "build": "tsc && vite build",
         "check-types": "tsc --noemit",
-        "eslint": "eslint src",
```

Comment:
this name-change is causing the code check failure

Reply:
I had to do this because the justfile doesn't like the shared name between eslint the bash command and eslint the pnpm script, i fixed it though, had to add a new eslint-path for a "bare" eslint command